apps/i18n/common/en_us.json

```diff
@@ -237,6 +237,7 @@
   "reportAbuse": "Report Abuse",
   "resetProgram": "Reset",
   "restoreThisVersion": "Restore this Version",
+  "reviewResponses": "5. Go to the assessment/survey tab for your section to review student responses.",
```

Comment:
Good point. It might make more sense to extract the numbering (and have it not be localized).